xla/stream_executor/gpu/gpu_stream.h

```diff
@@ -93,6 +96,10 @@ GpuStream* AsGpuStream(Stream* stream);
 // Extracts a GpuStreamHandle from a GpuStream-backed Stream object.
 GpuStreamHandle AsGpuStreamValue(Stream* stream);
 
+// Gets the priority range of streams.
+// Returns true if retrieval of range was successful and false otherwise.
+bool GetStreamPriorityRange(int* lowest, int* highest);
```

Comment:
Looks like this function is unused outside of this file, so can it be make private? Actually looks like it might go away once that code is moved to GpuDriver.

Reply:
Let me move this to GPUDriver too, i dont want to change the GPUDriver::CreateStream interface since it's already used by rocM and cuda. Both libraries assume priority as type int, so it should be safe to make this as a member function of GPUDriver.